frontend/src/components/review/CommentsPanel.tsx

```diff
@@ -115,25 +115,28 @@ const CommentsPanel = ({
     if (comments.length > 0) {
       return (
         <Box>
-          {comments.map(function (comment: CommentResponse, i: number) {
-            if (threadHeadMap[i]) {
-              return (
-                <CommentThread
-                  comments={comments}
-                  commentsIdx={i}
-                  key={comment.id}
-                  reviewerName={reviewerName}
-                  setComments={setComments}
-                  updateCommentsAsResolved={updateCommentsAsResolved}
-                  updateThreadHeadMap={updateThreadHeadMap}
-                  threadHeadMap={threadHeadMap}
-                  translatorId={translatorId}
-                  translatorName={translatorName}
-                />
-              );
-            }
-            return null;
-          })}
+          {
+            // eslint-disable-next-line func-names
+            comments.map(function (comment: CommentResponse, i: number) {
```

Comment:
hmmm would this error disappear if the function were an arrow function? eg `(comment: CommentResponse, i: number) 
 => {....`

Reply:
yes this seems to work: `{comments.map((comment: CommentResponse, i: number) => {` --would you prefer this?

Reply:
yeah lets do it!